### PR TITLE
Fix issues raised in PED

### PR DESCRIPTION
**Pull request by noobmaster19**
Resolved each issues w a corresponding commit for easy reference

### CODE REVIEW

**Review by kdeasymoneysniper — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -78,12 +79,6 @@ Orange text     | Headings and subheadings of various size
 
 1. Refer to the [Features](#features) below for details of each command.
 
-<div markdown="block" class="alert alert-info">
-:exclamation: Try not to edit the **data file** directly!  
-* This may lead to invalid values for data fields. 
-* In the event of this, *ComputingConnection* will restart with an empty database! 
-</div>
-
```
Comment (timothywongej) [inline docs/UserGuide.md:86]: Dion, did you pull from the master branch before making changes? A lot of the new edits were deleted in this PR.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -104,17 +99,10 @@ Understanding the structure of a **contact** in *ComputingConnection* is importa
 
 Category        | Specific fields | Valid items | Requirement
 ----------------|-----------------|-----------------|-----------------
-Personal data fields  | 1. `n/` : Name <br><br> 2. `e/` : Email |1. Alphanumeric <br><br> 2. Email Regex | Compulsory
-University data fields   | 3. `f/` : Faculty <br><br>  4. `m/` : Major | 3. NUS Faculties: <br> fass <br> business <br> computing <br> dentistry <br> sde <br> engineering <br> medicine <br> science <br> law <br><br> 4. Alphanumeric |Compulsory
-Skill data fields | 5. `s/` : Skill <br><br> 6. `l/` : Programming Language <br><br> 7. `fr/` : Framework | 5. Alphanumeric <br><br> 6. Alphanumeric and the '+' and '#' characters <br><br> 7. Alphanumeric and the '.' character| Optional
-Miscellaneous data fields| 8. `t/` : Tag <br><br> 9. `r/` : Remark <br><br> 10. `int/` : Interaction <br><br> 11. `compat/` : Compatability | 8. Alphanumeric, less than or equal to 30 characters <br><br> 9. Alphanumeric <br><br> 10. Alphanumeric, Date <br><br> 11. 0 - 100 | Optional
-
-<div markdown="block" class="alert alert-info">
```
Comment (timothywongej) [inline docs/UserGuide.md:112]: This info block was deleted as well.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -217,14 +207,14 @@ Commands that are related to a specific contact.
...
 
-Format: `add n/NAME e/EMAIL f/FACULTY m/MAJOR [compat/COMPATABILITY] [s/SKILL]…​ [l/LANGUAGE]…​ [fr/FRAMEWORK]…​ [t/TAG]…​ [r/REMARK]…​`
+Format: `add n/NAME e/EMAIL f/FACULTY m/MAJOR [compat/COMPATIBILITY] [s/SKILL]…​ [l/LANGUAGE]…​ [fr/FRAMEWORK]…​ [t/TAG]…​ [r/REMARK]…​`
 
 Examples: 
 
 * `add n/Timothy Wong e/timothy@nus.edu.sg f/computing m/computer science` 
   > Adds a person named 'Timothy Wong', with an email of 'timothy@nus.edu.sg', faculty of 'computing', and major of 'computer science'. 
-* `add n/Timothy Wong e/timothy@nus.edu.sg f/computing m/computer science compat/80 s/frontend l/javascript r/interest in web development` 
-  > Adds a person named 'Timothy Wong', with an email of 'timothy@nus.edu.sg', faculty of 'computing', and major of 'computer science', with a compatibility of 80/100, with skills 'frontend', languages 'javascript', and a remark of 'interest in web development'.
```
Comment (timothywongej) [inline docs/UserGuide.md:227]: Likewise for this edit

File: docs/UserGuide.md [hunk 4]
```diff
@@ -245,7 +235,7 @@ Format: `edit INDEX [n/NAME] [e/EMAIL] [f/FACULTY] [m/MAJOR] [compat/COMPATABILI
 
 <div markdown="block" class="alert alert-info">
 :exclamation: Editing is not cumulative!
-* When editing data fields, the existing items of the data field of the contact will be removed i.e adding of items is not cumulative (see [append](#appending-items-to-data-fields--append)).
```
Comment (timothywongej) [inline docs/UserGuide.md:248]: This change as well

File: docs/UserGuide.md [hunk 5]
```diff
@@ -325,7 +314,7 @@ Examples:
 * Index is based on current list displayed on left side of screen. 
 </div>
 
-##### Locating contacts by name : `find`
```
Comment (timothywongej) [inline docs/UserGuide.md:328]: Here too

File: docs/UserGuide.md [hunk 6]
```diff
@@ -315,7 +304,7 @@ Examples:
 *  `view 1` 
 > Displays the details of contact indexed at 1 in the currently displayed list.
 *  `view 2`
-> Displays the details of contact indexed at 2 in the currently displayed list.
```
Comment (timothywongej) [inline docs/UserGuide.md:318]: Here too

File: docs/UserGuide.md [hunk 7]
```diff
@@ -370,7 +359,7 @@ Examples:
 </div>
 
 ### Organisation-specific commands
-Commands that are related to organisations.
```
Comment (timothywongej) [inline docs/UserGuide.md:373]: Here as well

File: docs/UserGuide.md [hunk 8]
```diff
@@ -430,15 +419,7 @@ Examples:
...
 ### Future commands
-Commands to be implemented in future versions.
-
-##### Updates to data fields
-Data fields of a contact will be updated to meet the following specifications in future updates of *ComputingConnection*.
-
-Data field | Future updates
------------|---------------
-`f/` : Faculty | Case insensitive items, wider scope of NUS faculties
-`m/` : Major | Valid items to correspond with valid faculties
```
Comment (timothywongej) [inline docs/UserGuide.md:441]: This entire block as well

**Review by timothywongej — COMMENTED** [review]
Dion, I think you may not have pulled from master before making changes - quite a lot of changes were deleted. Reviewer maybe can take a look through next time.

File: src/main/java/seedu/address/logic/commands/RemoveCommand.java [hunk 9]
```diff
@@ -178,9 +178,7 @@ private static Person createRemovePerson(Person personToRemoveFrom, RemovePerson
 
         // Arrange previous skills in an array
         ArrayList<T> elementArray = new ArrayList<>();
-        elementArray.addAll(previousSet);
```
Comment (timothywongej) [inline src/main/java/seedu/address/logic/commands/RemoveCommand.java:181]: this deletion caused a bug....

**Pull request merged**